scripts/jest/spec-equivalence-reporter/equivalenceReporter.js

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+class EquivalenceReporter {
+  onTestCaseResult(test, testCaseResult) {
+    console.log(
+      `EQUIVALENCE: ${testCaseResult.title}, ` +
+        `status: ${testCaseResult.status}, ` +
+        `numExpectations: ${testCaseResult.numPassingAsserts}`
```

Comment:
In Jest 29, the testCaseResult.numPassingAsserts always returns 0. I submitted a PR https://github.com/facebook/jest/pull/13795 to implement the return of the actual value

Reply:
Thanks for the fix, pulling that update into React with https://github.com/facebook/react/pull/26150

Reply:
The fix for numPassingAsserts has been released in v29.4.1 😂